src/AmalgamatorSession.ts

```diff
@@ -675,6 +696,30 @@ export class AmalgamatorSession extends LoggingDebugSession {
             ].customRequest('cdt-gdb-adapter/Memory', args);
             response.body = childResponse.body;
             this.sendResponse(response);
+        } else if (command === 'cdt-amalgamator/resumeAll') {
+            const [, threads] = await this.collectChildTheads();
+            await Promise.all(
+                threads.map(async (thread) => {
+                    const [childIndex] = await this.getThreadInfo(thread.id);
+                    const childResponse = await this.childDaps[
+                        childIndex
+                    ].threadsRequest();
+                    const threadInfo = childResponse.body
+                        .threads[0] as ThreadInfo;
```

Comment:
When I use breakpoints to debug at childResponse.body, I always see that there is only one thread that is threads[0]. However, I have updated the code so that it can traverse all the threads that exist.

Reply:
I understand that, if you see only one thread ever it is because non of your childdaps are multi-threaded programs.